Documentation/network/clustermesh/mcsapi.rst

```diff
@@ -0,0 +1,165 @@
+.. _gs_clustermesh_mcsapi:
+
+*********************************
+Multi-Cluster Services API (Beta)
+*********************************
+
+.. include:: ../../beta.rst
+
+This tutorial will guide you to through the support of `Multi-Cluster Services API (MCS-API) <MCS_API_KEP>`_ in Cilium.
+
+.. _MCS_API_KEP: https://github.com/kubernetes/enhancements/blob/master/keps/sig-multicluster/1645-multi-cluster-services-api/README.md
```

Comment:
```suggestion
This tutorial will guide you to through the support of `Multi-Cluster Services API (MCS-API)`_ in Cilium.

.. _`Multi-Cluster Services API (MCS-API)`: https://github.com/kubernetes/enhancements/blob/master/keps/sig-multicluster/1645-multi-cluster-services-api/README.md
```

Otherwise, the link does not seem to be correctly taken into account.